public/app/core/components/Signup/SignupPage.tsx

```diff
@@ -108,7 +111,9 @@ export const SignupPage: FC<Props> = (props) => {
               </Field>
 
               <HorizontalGroup>
-                <Button type="submit">Submit</Button>
+                <Button type="submit" id="submit">
```

Comment:
```suggestion
                <Button type="submit">
```
ID is not necessary for buttons.

Reply:
addressed